utils/pi/services/Dockerfiles/Dockerfile-salmonmd-64

```diff
@@ -1,9 +1,8 @@
-FROM balenalib/raspberrypi4-64-debian-python:bookworm-build
+FROM balenalib/raspberrypi5-python:3.11-bookworm-build
 
-RUN apt-get update && apt-get install -y git ffmpeg
+RUN apt-get update && apt-get install -y git ffmpeg python3-opencv
 
-RUN python3 -m pip install -U pip
-RUN python3 -m pip install -U opencv-contrib-python-headless ffmpeg-python
+RUN python3 -m pip install -U pip && python3 -m pip install -U ffmpeg-python
```

Comment:
👍 I see this convo leading toward topics around how opencv is installed to support h264 (aka avc1) which linux distribution it's running on. https://stackoverflow.com/questions/70247344/save-video-in-opencv-with-h264-codec

Reply:
Yea. Though, I do feel like because the Raspberry Pi 5 is pretty new and they use the same OS (bookworm) as an updated Raspberry Pi 4, the pip package doesn't have a way to figure out whether the hardware codec exists in the raspi (h264_v4l2m2m). Raspberry Pi 5 removed that hardware encoding in the kernel because they found that the CPU is powerful enough to not require it anymore it seems, so that made it need a system specific install of OpenCV.

Reply:
Absolutely!